provider/internal/cdfmc/resource_test.go

```diff
@@ -0,0 +1,38 @@
+package cdfmc_test
+
+import (
+	"testing"
+
+	"github.com/CiscoDevnet/terraform-provider-cdo/internal/acctest"
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+)
+
+var resourceModel = struct {
+	Name string
+}{
+	Name: acctest.Env.CloudFmcResourceName(),
+}
+
+const resourceTemplate = `
+resource "cdo_cdfmc" "test" {
+}`
+
+var resourceConfig = acctest.MustParseTemplate(resourceTemplate, resourceModel)
+
+func TestAccCdFmcResource(t *testing.T) {
```

Comment:
I'd remove this test entirely; we are not going to add the ability to easily remove a cdFMC at any point, so no point keeping it in here.

Reply:
I see, but beside that this test will also come in handy for verifying any future changes manually.